.gitignore

```diff
@@ -2,7 +2,6 @@
 /.idea/
 .idea/
 *.iml
-*.class
```

Comment:
why is .class removed? These are binary files that are compiled separately on everyone's computer that don't need to be included